config/sample_application.yml

```diff
@@ -12,3 +12,5 @@ PUSHER_CLUSTER:
 TWITCH_CLIENT_ID:
 TWITCH_CLIENT_SECRET:
 TWITCH_WEBHOOK_SECRET:
+
+PERCY_TOKEN:
```

Comment:
is this needed only during CI testing? If so, we probably don't need it here

(ps. also missing a new line here)

Reply:
Ah, yep; it's only ever necessary for CI (unless you needed to debug Percy and run it locally, which you probably won't be doing unless you know what you're doing :)

I guess I added that a bit eagerly. I have removed it now 👍